<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomMapGenerator</class>
 <widget class="QDialog" name="RandomMapGenerator">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Map Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_icons.qrc">
    <normaloff>:/icons/openrpgmaker.png</normaloff>:/icons/openrpgmaker.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame
{
	border-width: 1px;
	border-style: solid;
	border-color: palette(window-text);
}

.QFrame&gt;QLabel
{
	border-width: 0px;
	background-color: palette(window-text);
	color: palette(window);
	font-weight: bold;
}

.QFrame&gt;QCheckBox
{
	background-color: palette(window-text);
	border-width: 0px;
	border-style: none;
	color: palette(window);
	spacing: 3px;
	font-weight: bold;
	padding: 0px 0px 0px 3px;
}

.QFrame&gt;QCheckBox::indicator
{
	min-width: 8px;
	max-width: 8px;
	min-height: 8px;
	max-height: 8px;
	width: 8px;
	height: 8px;
	border-width: 2px;
	border-radius: 0px;
	border-style: ridge;
	border-color: palette(mid);
}

.QFrame&gt;QCheckBox::indicator:unchecked
{
	background-color: palette(base);
}

.QFrame&gt;QCheckBox::indicator:checked
{
	background-color: palette(window-text);
}

.QFrame&gt;QCheckBox::indicator:hover
{
	border-color: palette(highlight);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_26">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_39">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Map Type</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="cboxMapType">
             <item>
              <property name="text">
               <string>Dungeon-Like Structure</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cavern-Like Structure</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mountain-Like Structure</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>World Map</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Town</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Building</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Maze</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbConfigureGenerator">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="gui_icons.qrc">
               <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbGenerateMaps">
             <property name="toolTip">
              <string>Generate Random Maps</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="gui_icons.qrc">
               <normaloff>:/icons/generaterandommap.png</normaloff>:/icons/generaterandommap.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frmRandomSeed">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Random Seed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="leRandomSeed">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbNewSeed">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="gui_icons.qrc">
               <normaloff>:/icons/newrandomseed.png</normaloff>:/icons/newrandomseed.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frmMapPreview">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Map Preview</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="cboxSelectMap">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Zoom:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbZoom1to1">
           <property name="toolTip">
            <string>1/1</string>
           </property>
           <property name="statusTip">
            <string>Zoom to a 1/1 ratio</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="gui_icons.qrc">
             <normaloff>:/icons/zoom1_1.png</normaloff>:/icons/zoom1_1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbZoom1to2">
           <property name="toolTip">
            <string>1/2</string>
           </property>
           <property name="statusTip">
            <string>Zoom to a 1/2 ratio</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="gui_icons.qrc">
             <normaloff>:/icons/zoom1_2.png</normaloff>:/icons/zoom1_2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbZoom1to4">
           <property name="toolTip">
            <string>1/4</string>
           </property>
           <property name="statusTip">
            <string>Zoom to a 1/4 ratio</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="gui_icons.qrc">
             <normaloff>:/icons/zoom1_4.png</normaloff>:/icons/zoom1_4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbZoom1to8">
           <property name="toolTip">
            <string>1/8</string>
           </property>
           <property name="statusTip">
            <string>Zoom to a 1/8 ratio</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="gui_icons.qrc">
             <normaloff>:/icons/zoom1_8.png</normaloff>:/icons/zoom1_8.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="StaticMapView" name="gvMapPreview">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StaticMapView</class>
   <extends>QGraphicsView</extends>
   <header>staticmapview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gui_icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RandomMapGenerator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RandomMapGenerator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
